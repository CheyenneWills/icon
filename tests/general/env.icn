procedure main()
   write()
   write(&date, " ", &clock, "  (", &dateline, ")")
   write(&version)
   every write("   ", &features)
   write("host: ", &host)
   write()
end
