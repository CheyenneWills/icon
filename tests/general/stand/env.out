#
# This test (and only this test) should show some differences.
#
